packages/timeline-state-resolver/src/devices/__tests__/vizMSE.spec.ts

```diff
@@ -848,9 +868,179 @@ describe('vizMSE', () => {
 		commandReceiver0.mockClear()
 		await mockTime.advanceTimeToTicks(21200)
 		expect(rundown.out).toHaveBeenCalledTimes(1)
-		expect(rundown.out).nthCalledWith(1, 1337, 'FULL1')
+		expect(rundown.out).nthCalledWith(1, {
+			vcpid: 1337,
+			channel: 'FULL1',
+		})
 		expect(rundown.take).toHaveBeenCalledTimes(0)
 
+		expect(onError).toHaveBeenCalledTimes(0)
+	})
+	test('vizMSE: Show lifecycle', async () => {
```

Comment:
This test contains slightly less than 30 assertions...
A good rule of thumb is to only have one assertions pr test (sometimes more is needed)
Keeping the number assertions pr test low allows for faster troubleshooting when a test fails. If a test fail that only have one assertion, we immediatly know exactly what is wrong in the code. If a test fails that have more than 5 assertions we then have to start debugging our test.

The maintenance burden is also a lot smaller by having many small tests, than few large tests :)